cypress/e2e/buy-shirt.cy.ts

```diff
@@ -1,26 +1,40 @@
+import {MenuContentPage} from "../page/index";
+import {LoginPage} from "../page/index";
+import {AddressStepPage} from "../page/index";
+import {PaymentStepPage} from "../page/index";
+import {ProductsListPage} from "../page/index";
+import {ShippingStepPage} from "../page/index";
+import {ShopingCartPage} from "../page/index";
+
+const menuContentPage = new MenuContentPage();
+const loginPage = new LoginPage();
+const addressStepPage = new AddressStepPage();
+const paymentStepPage = new PaymentStepPage();
+const productsListPage = new ProductsListPage();
+const shippingStepPage = new ShippingStepPage();
+const shopingCartPage = new ShopingCartPage();
+
 describe("Buy a t-shirt", () => {
-  it("then the t-shirt should be bought", () => {
-    cy.visit("http://automationpractice.com/");
-    cy.get("#block_top_menu > ul > li:nth-child(3) > a").click();
-    cy.get(
-        "#center_column a.button.ajax_add_to_cart_button.btn.btn-default",
-    ).click();
-    cy.get("[style*='display: block;'] .button-container > a").click();
-    cy.get(".cart_navigation span").click();
-
-    cy.get("#email").type("aperdomobo@gmail.com");
-    cy.get("#passwd").type("WorkshopProtractor");
-
-    cy.get("#SubmitLogin").click();
-    cy.get("[name='processAddress']").click();
-    cy.get("#cgv").click();
-    cy.get("[name='processCarrier']").click();
-    cy.get(".bankwire").click();
-    cy.get("#cart_navigation > [type='submit']").click();
-
-    cy.get("#center_column > div > p > strong").should(
-        "have.text",
-        "Your order on My Store is complete.",
-    );
+  it("then should be bought a t-shirt", () => {
+    menuContentPage.visitMenuContentPage();
+    menuContentPage.goToTShirtMenu();
+
+    shopingCartPage.addShoppingCart();
+    shopingCartPage.clickButtonContainer();
+    shopingCartPage.toLogin();
+
+    loginPage.putEmailUser();
+    loginPage.putPasswordUser();
+    loginPage.goToAddressCheckout();
+
+    addressStepPage.addressCheckout();
+
+    shippingStepPage.checkAgreement();
+    shippingStepPage.goToCarrier();
+
+    paymentStepPage.bankWireclick();
+
+    productsListPage.clickOncheckout();
```

Comment:
rename the method as confirmOrder or something similar, also, this method should be part of the paymentStepPage